src/Umbraco.Infrastructure/Migrations/MigrationPlanExecutor.cs

```diff
@@ -320,11 +335,43 @@ private void RunMigration(Type migrationType, MigrationContext context)
         {
             _rebuildCache = true;
         }
+
+        if (migration.InvalidateBackofficeUserAccess)
+        {
+            _invalidateBackofficeUserAccess = true;
+        }
     }
 
     private void RebuildCache()
     {
         _publishedSnapshotService.RebuildAll();
         _distributedCache.RefreshAllPublishedSnapshot();
     }
+
+    private async Task RevokeBackofficeTokens()
+    {
+        using IServiceScope scope = _serviceScopeFactory.CreateScope();
+
+        IOpenIddictApplicationManager openIddictApplicationManager = scope.ServiceProvider.GetRequiredService<IOpenIddictApplicationManager>();
+        var backOfficeClient = await openIddictApplicationManager.FindByClientIdAsync(Constants.OAuthClientIds.BackOffice);
```

Comment:
Not happy that we only invalidate our own client, the only way to make this extendable is to use the options pattern and loop over all the clients configured that way.
Might be overkill 🤷

Reply:
I think it's overkill. Users are tied to the core clients. We _could_ consider handling the Swagger and Postman clients too, but really... those are just for dev, let's not bother with them now.